modules/core/src/test/scala/gem/arb/Offset.scala

```diff
@@ -9,13 +9,15 @@ import org.scalacheck._
 import org.scalacheck.Arbitrary._
 
 trait ArbOffset {
-  import ArbAngle._
-
   implicit val arbOffsetP: Arbitrary[Offset.P] =
-    Arbitrary(arbitrary[Angle].map(Offset.P(_)))
+    Arbitrary(
+      Gen.chooseNum(0, 10000).map(mas => Offset.P(Angle.fromMilliarcseconds(mas)))
+    )
```

Comment:
This is unfortunate, but necessary since super large offsets don't fit in the `NUMERIC(9,3)` field we use for offsets.  We may want to revisit this because we can't test with negative offsets as it stands?